payments-core/src/main/java/com/stripe/android/payments/core/analytics/RealErrorReporter.kt

```diff
@@ -19,18 +20,14 @@ class RealErrorReporter @Inject constructor(
         stripeException: StripeException?,
         additionalNonPiiParams: Map<String, String>,
     ) {
-        val additionalParams = mapOf(
-            "analytics_value" to stripeException?.analyticsValue(),
-            "status_code" to stripeException?.statusCode?.toString(),
-            "request_id" to stripeException?.requestId,
-            "error_type" to stripeException?.stripeError?.type,
-            "error_code" to stripeException?.stripeError?.code,
-        ).plus(additionalNonPiiParams).filterNotNullValues()
+        val paramsFromStripeException = if (stripeException == null) {
+            emptyMap()
+        } else {
+            ErrorReporter.getAdditionalParamsFromStripeException(stripeException = stripeException)
+        }
+        val additionalParams = (paramsFromStripeException + additionalNonPiiParams).filterNotNullValues()
```

Comment:
There shouldn't be any null values here.

Reply:
removed `filterNotNullValues`